code/src/components/Album/Album.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import moment from 'moment';
+import './Album.css';
+import Artists from 'components/Artists/Artists';
+import AlbumImage from 'components/AlbumImage/AlbumImage';
```

Comment:
There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.

Reply:
Great to know! I was actually wondering about this